Assets/Scripts/GaugeBehavior.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class GaugeBehavior : MonoBehaviour
+{
+    public Image gauge;
+    public float lowPercentage = 0.2f;
+    private Animator animator;
+
+    private void Start()
+    {
+        animator = gameObject.GetComponent<Animator>();
+    }
+
+    public void SetVal(float capacity, float maxCapacity)
```

Comment:
can't you keep `maxCapacity` as a private variable instead of as a parameter here?